playbooks/utils/expand_disk.yml

```diff
@@ -24,6 +24,19 @@
       ansible.builtin.command: df -h
       register: starting_disk_size
 
+    - name: check the starting lv size
+      ansible.builtin.set_fact: 
+        starting_lv_size: "{{ starting_disk_size.stdout_lines[2].split()[1] }}"
+
+    - name: view the starting_disk_size
+      ansible.builtin.debug:
+        var: starting_lv_size
+
+    - name: End the play if the disk is expanded
+      ansible.builtin.meta: end_host
+      when:
+      - starting_lv_size == '28G'
```

Comment:
This is awesome! It will stop working correctly if we change the default size of the disk in future, but I think that's okay.

We could try doing something sneaky like checking the disk size in vSphere, but that is probably over the "just enough automation" line.